src/App.tsx

```diff
@@ -90,7 +90,7 @@ export class App extends React.Component<{}, State> {
             <AppBar position="relative">
                 <Toolbar>
                 <Typography variant="h6" color="inherit" noWrap>
-                    Pelm Demo
+                    Pelm Connect Demo
```

Comment:
I think Pelm Demo is more appropriate as we're also demo-ing the requests you can make.